buildSrc/private/src/main/kotlin/androidx/build/playground/VerifyPlaygroundGradleConfigurationTask.kt

```diff
@@ -52,7 +52,9 @@ abstract class VerifyPlaygroundGradleConfigurationTask : DefaultTask() {
     @TaskAction
     fun checkPlaygroundGradleConfiguration() {
         compareProperties()
-        compareGradleWrapperVersion()
+        // TODO: re-enable when https://github.com/gradle/gradle/issues/20778
```

Comment:
Can you also file / link a bug here against the github component so I can remember to come back to this?

Reply:
not sure what you mean, create an internal bug for the gradle bug?

Reply:
created b/232580974